portals/publisher/source/src/app/components/Apis/Details/Endpoints/EndpointOverview.jsx

```diff
@@ -626,204 +700,169 @@ function EndpointOverview(props) {
-                                                        )}
-                                                        label={(
-                                                            <Typography>
-                                                                <FormattedMessage
-                                                                    id={'Apis.Details.'
-                                                                        + 'Endpoints.EndpointOverview'
-                                                                        + '.production.endpoint.production.label'}
-                                                                    defaultMessage='Production Endpoint'
-                                                                />
-                                                            </Typography>
-                                                        )}
-                                                    />
                                                 )}
+                                            />
                                             <Collapse in={endpointCategory.prod}>
-                                                {endpointType.key === 'default'
-                                                    ? (
+                                                <ServiceEndpoint
+                                                    classes={classes}
+                                                    api={api}
+                                                    services={servicesList}
+                                                    category='production_endpoints'
+                                                    type=''
+                                                    setAdvancedConfigOpen={toggleAdvanceConfig}
+                                                    esCategory='production'
+                                                    setESConfigOpen={toggleEndpointSecurityConfig}
+                                                    name={<FormattedMessage
+                                                        id={'Apis.Details.Endpoints.'
+                                                            + 'EndpointOverview.production'
+                                                            + '.endpoint.production.header'}
+                                                        defaultMessage='Production Endpoint'
+                                                    />}
+                                                    editEndpoint={editEndpoint}
+                                                    endpointURL={getEndpoints
+                                                    (
+                                                        'production_endpoints'
+                                                    )}
+                                                    existingService={getService}
+                                                    editService={editService}
```

Comment:
What is the purpose of this `index` prop here , It has repeated in below as well

```jsx
index={0}
```

Reply:
the index prop is needed when it has load_balanced endpoints. This is where advanced configs and security apply on.  Anyway, this has only one endpoint. But in the future, if it has load_balanced endpoints we need this.